opensrp-app/src/main/java/org/smartregister/repository/EventClientRepository.java

```diff
@@ -2007,6 +2041,39 @@ public ColumnAttribute column() {
         }
     }
 
+    public boolean deleteClient(String baseEntityId) {
+        try {
+            int rowsAffected = getWritableDatabase().delete(clientTable.name(),
+                    client_column.baseEntityId.name()
+                            + " = ?",
+                    new String[]{baseEntityId});
+            if (rowsAffected > 0) {
+                return true;
+            }
+        } catch (Exception e) {
+            Timber.e(e);
+        }
+        return false;
+    }
+
+    public boolean deleteEventsByBaseEntityId(String baseEntityId, String eventType) {
+
+        try {
+            int rowsAffected = getWritableDatabase().delete(eventTable.name(),
+                    event_column.baseEntityId.name()
+                            + " = ? AND "
+                            + event_column.eventType.name()
+                            + " != ?",
```

Comment:
A bit of clarification here, the method says delete by baseEntityId, the params are baseEntityId and eventType yet this checks that `eventType != ` to the `eventType` passed as a param.

Reply:
Not sure, I did not change this method